### PR TITLE
Plat 212 support package level stacks

### PR DESCRIPTION
**Pull request by arran-standish**
For story https://jembiprojects.jira.com/browse/PLAT-212

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -34,3 +37,18 @@ services:
...
+networks:
+  openhim:
+    name: openhim_public
+    external: true
+  kafka:
+    name: kafka_public
+    external: true
+  elastic:
+    name: elastic_public
+    external: true
```
Comment (bradsawadye) [inline docker-compose.yml:54]:
```suggestion
    external: true

```

add new line at end of file

File: importer/analytics-datastore-elastic-search/docker-compose.config.yml [hunk 2]
```diff
@@ -22,3 +24,8 @@ configs:
     name: disi-es-index-config-fhir-report-patient.json-${disi_es_index_config_fhir_report_patient_json_DIGEST:?err}
     labels:
       name: disi
+
+networks:
+  elastic:
+    name: elastic_public
+    external: true
```
Comment (bradsawadye) [inline importer/analytics-datastore-elastic-search/docker-compose.config.yml:31]: above comment applies

File: importer/dashboard-visualiser-kibana/docker-compose.config.yml [hunk 3]
```diff
@@ -22,3 +24,8 @@ configs:
     name: disi-kibana-config-kibana-export.ndjson-${disi_kibana_config_kibana_export_ndjson_DIGEST:?err}
     labels:
       name: disi
+
+networks:
+  elastic:
+    name: elastic_public
+    external: true
```
Comment (bradsawadye) [inline importer/dashboard-visualiser-kibana/docker-compose.config.yml:31]: above comment applies

File: importer/hapi-fhir/docker-compose.config.yml [hunk 4]
```diff
@@ -18,3 +20,8 @@ configs:
     name: disi-hapi-fhir-fhirConfig.js-${disi_hapi_fhir_fhirConfig_js_DIGEST:?err}
     labels:
       name: disi
+
+networks:
+  hapi-fhir:
+    name: hapi-fhir_public
+    external: true
```
Comment (bradsawadye) [inline importer/hapi-fhir/docker-compose.config.yml:27]: above comment applies

File: importer/importer.sh [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      docker::deploy_config_importer $STACK_NAME "$compose_path" "${config_service_name}" "disi" &>/dev/null
+      overwrite "Waiting for config importer ${config_service_name} to run ... Done"
+    ) || {
+      log error "FATAL: Failed to deploy ${config_service_name}"
+    }
+    fi
+  else
+    log warn "Service '$TARGET' does not appear to be running... Skipping the deploy of ${config_service_name[*]}"
+  fi
+}
```
Comment (bradsawadye) [inline importer/importer.sh:47]: above comment applies

File: importer/openhim-core/docker-compose.config.yml [hunk 6]
```diff
@@ -32,3 +34,8 @@ configs:
     name: disi-openhim-config-openhim-import.json-${disi_openhim_config_openhim_import_json_DIGEST:?err}
     labels:
       name: disi
+
+networks:
+  openhim:
+    name: openhim_public
+    external: true
```
Comment (bradsawadye) [inline importer/openhim-core/docker-compose.config.yml:41]: above comment applies

File: importer/santedb-mpi/docker-compose.config.yml [hunk 7]
```diff
@@ -35,3 +37,8 @@ configs:
     name: disi-santempi-config-identityDomains.json-${disi_santempi_config_identityDomains_json_DIGEST:?err}
     labels:
       name: disi
+
+networks:
+  santedb:
+    name: santedb_public
+    external: true
```
Comment (bradsawadye) [inline importer/santedb-mpi/docker-compose.config.yml:44]: above comment applies

**Review by bradsawadye — CHANGES_REQUESTED** [review]
Looks good, just some minor things to address

File: package-metadata.json [hunk 8]
```diff
@@ -21,26 +22,33 @@
     "CLIENT_REGISTRY_HOST": "santedb-mpi",
     "CLIENT_REGISTRY_PORT": 8080,
     "CLIENT_REGISTRY_IDENTITY_DOMAIN": "http://ohie.org/nid",
+    "CLIENT_REGISTRY_STACKNAME": "santempi",
 
     "HAPI_FHIR_INSTANCES": "1",
     "FHIR_SERVER_URL": "http://hapi-fhir:8080",
     "FHIR_IG_URL": "https://openhie.github.io/hiv-ig",
+    "FHIR_CONFIG_IMPORTER_FOLDER_NAME": "hapi-fhir",
```
Comment (nour-borgi) [inline package-metadata.json:30]: This var is not used if I m not mistaken ?

**Review by nour-borgi — APPROVED** [review]
LGTM, one minor comment

**Review by bradsawadye — APPROVED** [review]

**Pull request merged**